katello.te

```diff
@@ -39,6 +39,8 @@ require{
     type passenger_t;
     type websockify_t;
     type httpd_t;
+    type candlepin_activemq_port_t;
```

Comment:
This assumes the type exists, otherwise it won't compile and users will also see errors during installation. I don't see the dependency in `katello-selinux` RPM, we need to followup with a patch that adds a Require statement.

I am assuming the port is defined in `candlepin-selinux` or similar external package, I haven't checked where it gets shipped: https://github.com/candlepin/candlepin/tree/master/server/selinux